src/boot/efi/util.c

```diff
@@ -166,6 +166,18 @@ EFI_STATUS efivar_get_raw(const EFI_GUID *vendor, const CHAR16 *name, CHAR8 **bu
         return err;
 }
 
+EFI_STATUS efivar_get_u8_boolean(const EFI_GUID *vendor, const CHAR16 *name, BOOLEAN *ret_value) {
```

Comment:
I think `u8` is misleading. Why not simply `efivar_get_boolean()`?

Reply:
I'll let you fight this one out with @poettering who suggested `efivar_get_u8_boolean`. I don't mind either one.